### ISSUE TITLE
Naming of linux build.

### ISSUE THREAD

**Issue by taitep** [deepnight/ldtk#1097]
You provide a build for linux, and there are multiple problems with the naming of the files. The .zip you downoad has "Ubuntu" in it, but it works on other distros too. And the AppImage is named installer, but really is just the executable.

**Comment by deepnight** [comment]
That's right, thanks for pointing that out. I have no clue what a good naming would be for a linux build: any suggestion?

**Comment by taitep** [comment]
just remove the ubuntu and insttaller parts. make sure that any time you remove ubuntu that linux also is in the name tho. if there is a file that includes ubuntu but not linux replace ubuntu with linux.

**Comment by taitep** [comment]
maybe this if you want to keep similar names and structure:
linux-distribution.zip (but probably include ldtk in the name here)
LDtk VERSION.AppImage

but i would probably just have the download directly provide the AppImage named somthing like this:
LDtk-VERSION.AppImage
If you do that you do not need to mention linux in the name because AppImage files are linux-specific.
If you really need compression, you could also use gzip, that comes preinstalled on most distros. Regular zip does too, but gzip can compress a single file directly instead of having it be the contents of the folder (so you dont have separate names and such) if you want to, so you could just have a download that is LDtk-VERSION.AppImage.gz and still get compression (if you want to change it up, i think that method is pretty convenient for the user but if you dont want to or cannot for some reason, the first set of names is my reccomendation).
